baseplate/context/__init__.py

```diff
@@ -27,7 +27,10 @@ def my_handler(self, context):
 from __future__ import print_function
 from __future__ import unicode_literals
 
-from ..core import BaseplateObserver
+from ..core import (
+    LocalSpan,
+    BaseplateObserver
```

Comment:
nit: trailing comma please (or just put 'em on one line which is fine if it's short enough)